sdcm/nemesis.py

```diff
@@ -4219,7 +4219,8 @@ def disrupt_add_remove_mv(self):
             try:
                 self.tester.create_materialized_view(ks_name, base_table_name, view_name, [column],
```

Comment:
why not put the filter here in the nemesis ? 
i.e. this failure seen in the log it tied to the nemesis flow (i.e. taking a node down)

I think it would be better to set the filter here ontop of the called function, and not add a parameter, and do that inside the function, this filtering isn't part of the function goals, it's part of the nemesis

also this function is being called only here, (and once more in test that isn't used)